kotlin/generate-offline-map/src/main/java/com/esri/arcgisruntime/sample/generateofflinemap/MainActivity.kt

```diff
@@ -116,8 +114,10 @@ class MainActivity : AppCompatActivity() {
           if (minPoint != null && maxPoint != null) {
             val envelope = Envelope(minPoint, maxPoint)
             downloadArea.geometry = envelope
+            // enable the map offline button only after the map is loaded
+            if (!takeMapOfflineButton.isEnabled) takeMapOfflineButton.isEnabled = true
+          }
           }
```

Comment:
strange indent, but it looks like this is the only thing the reformatter picks up.